### ISSUE TITLE
[ACTIVE_PREQUAL_ARBITRATION_WIN] SWARM-1857 — PR1017 current-main integration topology

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1857]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SCIENTIFIC_CONTROL: #548
SWARM_WORKER_ID: SWARM-1857
STATUS: ACTIVE_NONOWNING_INTEGRATION_PREQUAL_ARBITRATION_WIN
SWARM_LANE_KEY: AUDIT-B|PR1017|CURRENT-MAIN-INTEGRATION-PREQUAL|EXACTHEAD-A54AE309|V1
PREFERRED_LANE: #14 AUDIT-B — Data / Infra / Compute / Release Auditor
PREFERRED_WORK_KIND: INTEGRATION_CONVERGENCE
SELECTED_WORK_KIND: READ_ONLY_INTEGRATION_PREQUAL / CURRENT-MAIN-TOPOLOGY / DATA-INTEGRITY
PARENT_ISSUES: #4 #14 #548 #723 #916 #925 #958 #1017 #1755 #1760 #1768 #1771
BASE_SHA: 2da11c62f066671a1122888ec116242502f4fb9c
TARGET_PR: #1017
TARGET_EXACT_HEAD: a54ae30943ba3f25a2c1e7e5bf6b42c5ca89a1a3
TARGET_DECLARED_BASE_SHA: 58711afeadf7f29c107188cb15d1e73c728768bb
EXACT_HEAD_SHARED_CI: 34792020724=SUCCESS
INDEPENDENT_AUDIT: #1771 RELEASED_PASS / PASS_FOR_INTEGRATION_TERMINAL_RADA_REPLAY_EXACT_HEAD
AUDITED_MAIN_SNAPSHOT: 2da11c62f066671a1122888ec116242502f4fb9c
CURRENT_INTEGRATION_OWNER: #1760 ACTIVE under declared lease through 2026-09-14T06:05:31Z UTC unless terminal/released sooner; MERGE AUTHORITY NOT CLAIMED HERE
DOWNSTREAM_EXECUTION_OWNER: #1768 PR1491 ACTIVE / DISTINCT PRODUCT SEMANTIC
CLAIMED_AT_UTC: 2026-09-14T04:16:38Z
LEASE_UNTIL_UTC: 2026-09-14T10:16:38Z unless terminal/released sooner
LOCAL_FREE_ONLY: true
PRODUCT_MUTATION: false
AUDIT_PRODUCT_PATCH_AUTHORITY: false
MERGE_AUTHORITY: false
TRAINING_CLAIM: false

POSTCLAIM DIRECT ARBITRATION — WIN:
- direct OPEN issue collection newest-first at 100/page: pages 1..13 nonempty; page 14 exact `[]` sentinel;
- direct OPEN PR collection newest-first at 100/page: pages 1..6 nonempty; page 7 exact `[]` sentinel;
- exact-key race refresh after collection returns only SWARM-1857 for `AUDIT-B|PR1017|CURRENT-MAIN-INTEGRATION-PREQUAL|EXACTHEAD-A54AE309|V1`;
- no earlier valid same-semantic claimant found;
- live main race refresh remains exact `2da11c62f066671a1122888ec116242502f4fb9c`;
- PR1017 race refresh remains OPEN/draft/mergeable=true at exact `a54ae30943ba3f25a2c1e7e5bf6b42c5ca89a1a3`, four changed files;
- #1771 remains RELEASED_PASS; its terminal comment explicitly binds the PASS to `AUDITED_MAIN_SNAPSHOT=2da11c62...`, with 0 HIGH / 0 MEDIUM, exact-head CI 34792020724 SUCCESS and audit ownership released;
- #1760 remains active integration owner and already has PR1017 listed as a terminal integration candidate; this worker will hand topology only, not race merge;
- #1768 remains active distinct PR1491 expanded-V9 owner.

CANDIDATES COMPARED BEFORE REGISTRATION:
1. Direct central terminal-green intake — highest authority leverage but live-owned by #1760; YIELD.
2. PR1800 current-main integration prequalification — high D03 leverage but live-owned by earlier #1855; YIELD.
3. PR1017 current-main integration topology — SELECTED: terminal real LOCAL_FREE Rada replay, 4,619 survivors / 858,698,060 UTF-8 bytes, independently audited exact cleanup head, directly upstream of active expanded-V9 #1768.
4. PR1816 terminal R01 integration topology — backup only if PR1017 became unavailable.

WHY THIS PACKAGE:
PR1017 already completed the expensive real execution and has a fresh different-worker exact-head PASS against the same live main snapshot, but its PR base is historical main `58711afe...` and it remains draft. CI policy forbids gratuitous ancestry chasing yet requires a bridge if current-main changes overlap owned/dependency/test/workflow/authority surfaces or integration uncertainty remains. The active integrator should receive a concrete late-bound topology verdict rather than redo this analysis during intake.

OWNED SURFACES:
- read-only exact PR1017/current-main integration topology analysis;
- rebind exact PR head/state/draft/mergeability, exact-head shared CI and #1771 terminal verdict;
- inspect exact four-file PR delta, main-since-tested-base overlap, D03 dependency/authority evolution, current PR1491 dependency relationship and whether unchanged-head intake is safe under validation-head-stability policy;
- publish only `PREQUAL_READY_FOR_INTEGRATOR_CONSUMPTION`, `PREQUAL_BRIDGE_REQUIRED`, or `PREQUAL_HOLD` with exact blocker to #1760 and PR1017;
- no merge, Product code edit, head movement, CI rerun, corpus credit, downstream V9 execution, tokenizer fit or training.

AVOID_SURFACES:
- no mutation or audit takeover of PR1017 Product bytes/audit semantic (#1771 already terminal/released);
- no mutation of PR1491/#1768 expanded-V9, PR1459 matcher, PR940/PR1836 handoff, PR1821 decontamination, PR945 balance, tokenizer/packing/unique-loss/model/trainer/evaluation/final-test surfaces;
- no duplicate central integration claim while #1760 lease is active;
- no paid compute, foreign pretrained weights, external LLM/API corpus intelligence or final-test access.

OBJECTIVE:
Determine, with exact current Git topology and dependency evidence, the minimum lawful integration action for PR1017: unchanged-head intake, one late bridge + fresh exact-head CI, or HOLD. Hand that decision to the active integrator while preserving all zero-credit scientific boundaries.

PACKAGE_DIMENSIONS:
- implementation_or_primary_research (integration topology qualification)
- adversarial_or_negative_tests (overlap/dependency/authority conflict search)
- live_authority_binding
- end_to_end_or_integration_proof
- machine_readable/exact Git SHA + CI evidence binding
- operator/integrator handoff

ACCEPTANCE:
1. COMPLETED: postclaim direct OPEN issue + separate OPEN PR collections to physical empty sentinels; arbitration WIN.
2. Immediately before verdict rebind live main, PR1017 exact head/state/draft/mergeability, #1771 verdict, #1760 lease/ownership and #1768 downstream ownership.
3. Exact-head CI SHA/result must remain applicable to current head; head movement invalidates prior authority.
4. Compare current main against PR1017's tested base for overlap with the four owned paths plus declared behavior-bearing dependencies, tests, shared workflow and scientific authority roots. Main movement alone is not a bridge reason.
5. Explicitly distinguish GitHub mergeability from scientific/dependency compatibility; any unresolved authority overlap => bridge/HOLD, not optimistic intake.
6. Check whether PR draft state itself is an intake blocker and identify the lawful transition owner; do not mutate another owner's PR state here.
7. No positive corpus/training authority follows merely from integration readiness. Preserve exact fail-closed truth.

FALLBACK:
Concrete integration blocker => publish exact handoff to #1760/#1768, release this semantic, then rotate if execution budget remains.

TRUTH_BOUNDARY:
REAL_RADA_NO_SITE_REPLAY_EXECUTED=true
CANONICAL_RADA_SURVIVOR_AUTHORITY=false until lawful integration/authority promotion
CANONICAL_CAPACITY_CREDITED=0
TRAINING_AUTHORIZED_BYTES=0
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
UNIQUE_CAUSAL_LOSS_POSITIONS_AUTHORIZED=0
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false

NEXT NOW:
Exact changed-file/current-main overlap + dependency/authority topology analysis; then publish terminal prequalification and release.
